docs/introduction/1-welcome.mdx

```diff
@@ -61,4 +61,4 @@ If you feel another member of the community violated our CoC or you are experien
 
 ## About SumUp
 
-[SumUp](https://sumup.com) is a mobile-point of sale provider. It is our mission to make easy and fast card payments a reality across the _entire_ world. You can pay with SumUp in more than 30 countries, already. Our engineers work in Berlin, Cologne, Sofia, and Sāo Paulo. We write code in JavaScript, Swift, Ruby, Elixir, Erlang, and much more. Want to come and work with us? [Head to our careers page](https://sumup.com/careers) to find out more.
+[SumUp](https://sumup.com) is a mobile-point of sale provider. It is our mission to make easy and fast card payments a reality across the _entire_ world. You can pay with SumUp in more than 30 countries, already. Our engineers work in Berlin, Copenhagen, Kiev, Sofia, and Sāo Paulo. We write code in TypeScript, Swift, Go, Elixir, Erlang, and much more. Want to come and work with us? [Head to our careers page](https://sumup.com/careers) to find out more.
```

Comment:
🆕 🤩 

Maybe we can also update this in the OSS template if this is stiff a thing?

Reply:
Honestly, I think this should be rewritten completely. I just wanted to fix the most obvious errors 🤷‍♂️

Reply:
Makes sense!